### DISCUSSION TITLE
geoviews? species distribution

### DISCUSSION THREAD

**Discussion by VictoriAragon** [earthlab-education/Earth-Analytics-AY24#518] [Q&A]
On notebook 33 within Species Distribution, I get the error 

ImportError: Geographic projection support requires: geoviews."

Tried to import geoviews thinking it was a package I may have missed importing. But it doesnt import. 

Does anyone know why I am getting this error? How I can fix it?

**Comment by allenmoench** [comment]
I'm getting the same error![image](https://github.com/user-attachments/assets/3f1eab20-a598-48a3-81f8-74e3589f9516)

**Comment by lauren-pease** [comment]
@VictoriAragon First you'll want to install the geoviews package then import it: 

```python
%pip install geoviews
```

```python
import geoviews as gv
```

**Reply by VictoriAragon** [reply]
@lauren-alexandra thank you lauren this was helpful.

**Comment by eculler** [comment]
Hi folks -- we're finding that installing geoviews with pip is not always working! Check out the answer to this discussion post for a more consistent solution: https://github.com/earthlab-education/Earth-Analytics-AY24/discussions/449
